dev/generate_bom.py

```diff
@@ -155,9 +155,15 @@ def write_docker_version_files(self):
     """
     for comp in self.__component_versions:
       gradle_version = self.__version_from_tag(comp)
-      docker_tag = '{reg}/{comp}:{tag}'.format(reg=self.__docker_registry,
-                                               comp=comp,
-                                               tag=gradle_version)
+      docker_tag = ''
+      if comp == 'spinnaker-monitoring':
+        docker_tag = '{reg}/{comp}-daemon:{tag}'.format(reg=self.__docker_registry,
+                                                        comp=comp,
```

Comment:
It's not in here yet (but probably should be soon). Halyard sends the CLI a command describing how to install the dashboards, which (for now) only involves pulling a debian and running the correct script. In the future, I'd imagine that if the monitoring is being setup on a remote k8s cluster, it would use a "run job" stage to create/update the dashboards.